index.js

```diff
@@ -6,7 +6,12 @@ hexo.config.uglify = Object.assign({
   mangle: true,
   output: {},
   compress: {},
-  exclude: '*.min.js'
+  exclude: '*.min.js',
+  es6: false
```

Comment:
need to document new option. also mention "Minify JavaScript files with UglifyJS or Terser."

Reply:
Added.